src/provider.c

```diff
@@ -1176,10 +1177,35 @@ static const OSSL_ALGORITHM p11prov_decoders[] = {
     { NULL, NULL, NULL }
 };
 
+const char *p11prov_block_ops_names[OSSL_OP__HIGHEST + 1] = {
```

Comment:
Add `static`?

Reply:
Yeah, I guess I can declare it static, it won't be useful outside this file anyway.